ci: main red — playgrounds/blog + playgrounds/dashboard import gitignored .gen package

## Summary

`main` CI is red since #149 (blog playground) and #150 (dashboard playground) merged. Both playgrounds import a generated `.gen` package that is `.gitignore`'d, so `go vet` and `golangci-lint` cannot resolve it on CI.

## Failing runs

- Run [25160105707](https://github.com/binsarjr/sveltego/actions/runs/25160105707) (push to main, after #150)
- Run [25159969295](https://github.com/binsarjr/sveltego/actions/runs/25159969295) (push to main, after #149) — same error

## Error (exact log line)

```
playgrounds/blog/cmd/app/main.go:13:2: module github.com/binsarjr/sveltego@latest found
  (v0.0.0-00010101000000-000000000000, replaced by ./packages/sveltego),
  but does not contain package github.com/binsarjr/sveltego/playgrounds/blog/.gen
```

`golangci-lint` shows the parallel error:

```
buildssa: failed to load package : could not load export data:
  no export data for "github.com/binsarjr/sveltego/playgrounds/blog/.gen"
```

`isolated-modules (GOWORK=off)` and all `lint-and-test` matrix legs that touch playgrounds are red.

## Root cause

`playgrounds/blog/cmd/app/main.go` and `playgrounds/dashboard/cmd/app/main.go` both `import gen "github.com/binsarjr/sveltego/playgrounds/<name>/.gen"`. The `.gen/` directory is gitignored (see `.gitignore`), and the CI workflow does not generate it before `go vet`/`golangci-lint` runs.

The pre-existing `playgrounds/basic` is exempted in `.github/workflows/ci.yml` via:

```bash
case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
```

That skip was not extended for `blog` and `dashboard` when they landed.

## Fix options (pick one)

1. **Extend the skip list** in `.github/workflows/ci.yml` to cover `playgrounds/blog` and `playgrounds/dashboard` (matches existing pattern, lowest risk).
2. **Generate `.gen` in CI** before vet/lint — adds a sveltego-build step but actually exercises the generated code.
3. **Commit a stub `.gen/` per playground** with a generated marker — wrong direction (we ignore generated output for a reason).

Recommend option 1 short-term, then file follow-up for option 2 once codegen has a stable CLI entry point.

## Acceptance

- `gh run list --workflow ci --branch main --limit 1` returns `success`.
- All `lint-and-test` matrix legs green on push to main.
- `isolated-modules (GOWORK=off)` green.
- No regression of bench gate.

## References

- PR #149 (blog playground)
- PR #150 (dashboard playground)
- `.github/workflows/ci.yml` (skip block to amend)
- `.gitignore` (`.gen/` rule)

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -166,7 +166,7 @@ jobs:
           PATH="$(go env GOPATH)/bin:$PATH"
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -211,7 +211,7 @@ jobs:
           set -euo pipefail
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -257,7 +257,7 @@ jobs:
           mkdir -p coverage
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -306,7 +306,7 @@ jobs:
           set -euo pipefail
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -426,7 +426,7 @@ jobs:
           PATH="$(go env GOPATH)/bin:$PATH"
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -471,7 +471,7 @@ jobs:
           set -euo pipefail
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -517,7 +517,7 @@ jobs:
           mkdir -p coverage
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -566,7 +566,7 @@ jobs:
           set -euo pipefail
           dirs=$(go list -m -f '{{.Dir}}')
           for d in $dirs; do
-            case "$d" in *[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in *[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             if [[ "$SHARED" != "true" ]]; then
               case "$d" in
                 */packages/sveltego)             [[ "$SVELTEGO" == "true" ]]            || continue ;;
@@ -634,7 +634,7 @@ jobs:
           fi
           for m in $mods; do
             d=$(dirname "$m")
-            case "$d" in playgrounds/basic|*[/\\]playgrounds[/\\]basic) continue ;; esac
+            case "$d" in playgrounds/basic|playgrounds/blog|playgrounds/dashboard|*[/\\]playgrounds[/\\]basic|*[/\\]playgrounds[/\\]blog|*[/\\]playgrounds[/\\]dashboard) continue ;; esac
             echo "::group::isolated $d"
             (cd "$d" && go mod download && go build ./... && go test -race -short ./...)
             echo "::endgroup::"

